tensorpack/utils/viz.py

```diff
@@ -242,6 +243,34 @@ def dump_dataflow_images(df, index=0, batched=True,
                 vizlist = vizlist[vizsize:]
 
 
+def intensity_to_rgb(intensity, cmap='jet', normalize=False):
```

Comment:
Wow thanks for pointing it out! I'm now using cubehelix as default.
Blues might be a boring default 😄